<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conversion</class>
 <widget class="QWidget" name="Conversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conversion</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Amplitude</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>RMS</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Power</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Peak-to-peak</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="edt_dbm"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="edt_watts"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="edt_volt_rms"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="edt_volt_p"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="edt_volt_pp"/>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>dBm</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Watts</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Volts</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Volts</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Volts</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>resources/logo.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculateds the Power in [dBm] or [Watts] and Voltage in RMS, peak, peak-to-peak for sinusoidal signals in 50-Ohm systems. </string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
